commit 67a2f42b011d95e856210cbe662252234bd9d5a4 (refs/changes/04/1404/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-02T13:07:27-08:00 (2 years, 8 months ago)
    
    TOOLS-1673 sdcadm getAgentImages not working due to missing name attribute
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/lib/steps/agent-services.js b/lib/steps/agent-services.js
index 38e8c70..afd2df0 100644
--- a/lib/steps/agent-services.js
+++ b/lib/steps/agent-services.js
@@ -73,6 +73,7 @@ function agentServicesEnsureCreated(arg, cb) {
     agentNames.forEach(function (n) {
         var logLevelKey = n.toUpperCase().replace('-', '_') + '_LOG_LEVEL';
         agentServices[n] = {
+            name: n,
             type: 'agent',
             params: {
                 tags: {
